### PR TITLE
categories new method and view with form partial

### PR DESCRIPTION
**Pull request by bennfos**
check me bb

### CODE REVIEW

File: app/views/admin/categories/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <dl><%= form.text_field :name, class: "form-control" %></dl>
+  </dl>
+  <%= form.submit "#{verb} category", class: "btn btn-primary mt-3" %>
+<% end %>
+
+<% if method == :patch %>
+  <div class="js-media-form">
+    <%= render partial: "admin/pieces/media", locals: { model: model } %>
+  </div>
+<% end %>
```
Comment (mikekavouras) [inline app/views/admin/categories/_form.html.erb]: I don't think you need this!

Comment (bennfos) [reply app/views/admin/categories/_form.html.erb]: true. no media for categories

File: db/schema.rb [hunk 2]
```diff
@@ -36,16 +36,6 @@
...
-  create_table "assets", force: :cascade do |t|
-    t.integer "piece_id"
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
-    t.string "asset_file_name"
-    t.string "asset_content_type"
-    t.bigint "asset_file_size"
-    t.datetime "asset_updated_at"
-  end
-
```
Comment (mikekavouras) [inline db/schema.rb]: BYE BYE

Comment (bennfos) [reply db/schema.rb]: ???

Comment (bennfos) [reply db/schema.rb]: dunno what happened here

File: app/controllers/admin/categories_controller.rb [hunk 3]
```diff
@@ -10,6 +10,19 @@ def show
...
     end
 
+    def new
+      render "admin/categories/new", locals: {
+        category: Category.new
+      }
+    end
+
+    def create
+      category_params = params.require(:category).permit(:name)
```
Comment (mikekavouras) [inline app/controllers/admin/categories_controller.rb]: Organization nit: it's generally nice to tuck this into a private method like we do in [`PiecesController`](https://github.com/mikekavouras-organize/art/blob/master/app/controllers/admin/pieces_controller.rb#L52)

File: app/controllers/admin/categories_controller.rb [hunk 4]
```diff
@@ -10,6 +10,19 @@ def show
...
+      @category =  Category.new(category_params)
+      @category.save
```
Comment (mikekavouras) [inline app/controllers/admin/categories_controller.rb]: You can combine these two lines into `Category.create(category_params)`

File: app/controllers/admin/categories_controller.rb (hunk 3 above)
Comment (bennfos) [reply app/controllers/admin/categories_controller.rb]: yes, daddy

File: app/controllers/admin/categories_controller.rb (hunk 4 above)
Comment (bennfos) [reply app/controllers/admin/categories_controller.rb]: right away, dad

**Pull request merged**